Gemfile

```diff
@@ -12,14 +12,16 @@ def location_for(place, version = nil)
   end
 end
 
-group :development, :unit_tests do
-  gem 'json',                      :require => false
-  gem 'metadata-json-lint',        :require => false
-  gem 'puppet_facts',              :require => false
-  gem 'puppet-blacksmith',         :require => false
-  gem 'puppetlabs_spec_helper',    :require => false
-  gem 'rspec-puppet', '>= 2.3.2',  :require => false
-  gem 'simplecov',                 :require => false
+group :development do
+  gem 'pry',  :require => false
+end
+group :unit_tests do
+  gem 'metadata-json-lint',                :require => false
+  gem 'puppet_facts',                      :require => false
+  gem 'puppet-blacksmith',                 :require => false
+  gem 'puppetlabs_spec_helper', '~> 1.1',  :require => false
+  gem 'rspec-puppet', '>= 2.3.2',          :require => false
```

Comment:
The main distinction is that we want to update the travis.yml so that interactive tools are not installed for running the unit tests. I would not expect anyone working on a module to bundle install --without=unit_tests.

I'll update the commit's description once this all becomes ready.